### PR TITLE
Salt Day 11

### PR DESCRIPTION
**Pull request by Parzivus**
Would be good if this one can get looked over in the next 24 hours or so, I'm gonna be gone over Friday and the weekend so I won't be able to edit it.

### CODE REVIEW

File: script/Arcueid/Day 10/10_00_ARC10_5E.txt [hunk 1]
```diff
@@ -250,6 +250,7 @@ It ain't free to feed you people, you know."
 [sha:42f67e9971501c89138e1f3629aebfb9048fd242]{
 -- Page 642, Offset 25983.
 -- 真横から、流暢だが乱暴な日本語が聞こえた。
+// Thoughts?
```
Comment (rschlaikjer) [inline script/Arcueid/Day 10/10_00_ARC10_5E.txt:253]: All the edits in this file look like mistakes?

File: script/Arcueid/Day 11/11_00_ARC11_1B.txt [hunk 2]
```diff
@@ -72,7 +72,7 @@ I should let just it all go.
 [sha:803c0c8be2764a5f51450476350654c139d6f6f7]{
 -- Page 34, Offset 29579.
 -- ここで目を閉じるのが、まっとうな人間らしい選択だ。
-Pretending none of it ever happened would be the proper, human, choice to make.
+Pretending none of it ever happened would be the proper, human choice to make.
```
Comment (rschlaikjer) [inline script/Arcueid/Day 11/11_00_ARC11_1B.txt:75]: Kinda like the comma here to emphasize human

Comment (IrishKnight0) [reply script/Arcueid/Day 11/11_00_ARC11_1B.txt:75]: Maybe put italics instead? :/

File: script/Arcueid/Day 11/11_00_ARC11_3.txt [hunk 3]
```diff
@@ -1450,7 +1450,7 @@ Ah. I'm sorry, I'm all too aware of that myself.
 -- Page 247, Offset 30222.
 -- 「……でも、可笑しいね。殺した男と殺された女が、こうして抱き合ってるなんて」
 // boy and girl for a maybe failed "boy meets girl" allusion? this is kinda cheesy in a funny way i like it
-"But... It's funny isn't it? Killer boy and victim girl embracing each other like this?"
+"But... it's funny, isn't it? Killer boy and victim girl embracing each other like this?"
```
Comment (IrishKnight0) [inline script/Arcueid/Day 11/11_00_ARC11_3.txt:1453]: I think `isn't it?` should be without a question mark.

File: script/Arcueid/Day 10/10_00_ARC10_5E.txt (hunk 1 above)
Comment (Parzivus) [reply script/Arcueid/Day 10/10_00_ARC10_5E.txt:253]: This one was not intended to be uploaded :Aichanteehee:

File: script/Arcueid/Day 11/11_00_ARC11_3.txt (hunk 3 above)
Comment (Parzivus) [reply script/Arcueid/Day 11/11_00_ARC11_3.txt:1453]: I dunno, I kinda like this one. Makes her sound a little wistful.

File: script/Arcueid/Day 11/11_00_ARC11_2.txt [hunk 4]
```diff
@@ -404,8 +404,9 @@ I mean of course the vampiric impulse―――an innate desire, a %{i}craving%{/
 [sha:71a679bfce54efc4cd68983c9fe3d4028dc00bb8]{
 -- Page 64, Offset 29683.
 -- 原因は生命体としてのスケールの違いとも、彼らが血液を魂として捉えている、とも言われています」
-Who can say whether it's down to the sheer difference in ability between us as species,
- or whether by sucking someone's blood they steal their very soul."
+// Rephrased to avoid using "who can say" again
```
Comment (rschlaikjer) [inline script/Arcueid/Day 11/11_00_ARC11_2.txt:407]: Remember to delete comments that only make sense in the context of the PR before merging

**Review by rschlaikjer — APPROVED** [review]

**Pull request merged**